cypress/e2e/loginFormTest.cy.js

```diff
@@ -0,0 +1,43 @@
+describe('User Login Functionality', () => {
+  beforeEach(() => {
+    cy.visit('http://127.0.0.1:5500/social-media-client/index.html')
```

Comment:
```suggestion
    cy.visit('/')
```
Cypress hosts it's own local server, so you can make this much more simple